src/process/status.cr

```diff
@@ -287,13 +292,38 @@ class Process::Status
     io << "]"
   end
 
+  private def name_for_win32_exit_status
+    case @exit_status
+    # Ignoring LibC::STATUS_SUCCESS here because we prefer its numerical representation `0`
+    when LibC::STATUS_FATAL_APP_EXIT          then "STATUS_FATAL_APP_EXIT"
+    when LibC::STATUS_DATATYPE_MISALIGNMENT   then "STATUS_DATATYPE_MISALIGNMENT"
+    when LibC::STATUS_BREAKPOINT              then "STATUS_BREAKPOINT"
+    when LibC::STATUS_ACCESS_VIOLATION        then "STATUS_ACCESS_VIOLATION"
+    when LibC::STATUS_ILLEGAL_INSTRUCTION     then "STATUS_ILLEGAL_INSTRUCTION"
+    when LibC::STATUS_FLOAT_DIVIDE_BY_ZERO    then "STATUS_FLOAT_DIVIDE_BY_ZERO"
+    when LibC::STATUS_FLOAT_INEXACT_RESULT    then "STATUS_FLOAT_INEXACT_RESULT"
+    when LibC::STATUS_FLOAT_INVALID_OPERATION then "STATUS_FLOAT_INVALID_OPERATION"
+    when LibC::STATUS_FLOAT_OVERFLOW          then "STATUS_FLOAT_OVERFLOW"
+    when LibC::STATUS_FLOAT_UNDERFLOW         then "STATUS_FLOAT_UNDERFLOW"
+    when LibC::STATUS_PRIVILEGED_INSTRUCTION  then "STATUS_PRIVILEGED_INSTRUCTION"
+    when LibC::STATUS_STACK_OVERFLOW          then "STATUS_STACK_OVERFLOW"
+    when LibC::STATUS_CANCELLED               then "STATUS_CANCELLED"
+    when LibC::STATUS_CONTROL_C_EXIT          then "STATUS_CONTROL_C_EXIT"
+    end
```

Comment:
Would it be a good idea to protect this from typos using a macro loop (with `stringify` involved)